doc/bridge_packages/flame_forge2d/joints.md

```diff
@@ -60,10 +60,56 @@ It can for example be useful when simulating "soft-bodies".
 ```
 
 `ConstantVolumeJointDef` requires at least 3 bodies to be added using the `addBody` method.
+It also has two optional parameters:
 
-Optional param `frequencyHz` defines the frequency of oscillation of the joint.
-If it's not 0, the higher the value is, the less springy each of the compound `DistantJoint`s are.
+- `frequencyHz`: This parameter sets the frequency of oscillation of the joint.
+  If it is not set to 0, the higher the value,
+the less springy each of the compound `DistantJoint`s are.
 
-Another optional parameter is `dampingRatio`, it defines how fast the oscillation comes to rest.
-It takes values from 0 to 1, where 0 = no damping, 1 = critical damping.
+- `dampingRatio`: This parameter defines how quickly the oscillation comes to rest.
+  It ranges from 0 to 1, where 0 means no damping and 1 indicates critical damping.
 
+
+### `DistanceJoint`
+
+A `DistanceJoint` constrains two points on two bodies to remain at a fixed distance from each other.
+
+You can view this as a massless, rigid rod.
+
+```{flutter-app}
+:sources: ../../examples
+:page: distance_joint
+:show: widget code infobox
+:width: 200
+:height: 200
+```
+
+```dart
+    final distanceJointDef = DistanceJointDef()
+      ..initialize(firstBody, secondBody, firstBody.worldCenter, secondBody.worldCenter)
+      ..length = 10
+      ..frequencyHz = 3
+      ..dampingRatio = 0.2;
+
+    world.createJoint(DistanceJoint(distanceJointDef));
+```
+
+To create a `DistanceJointDef`, you can use the `initialize` method,
+which requires two bodies and a world anchor point on each body.
+The definition uses local anchor points, allowing for a slight violation of the constraint
+in the initial configuration. This is useful when saving and loading a game.
+
+The `DistanceJointDef` has three optional parameters that you can set:
+
+- `length`: This parameter determines the distance between
+the two anchor points and must be greater than 0.
```

Comment:
Interesting that this one is optional, what happens if it is not defined?

Reply:
It's inited with a default value. Added a note about that.